src/bin/ion/hex_reader.rs

```diff
@@ -120,54 +119,60 @@ impl<R: Read> Read for HexReader<R> {
     }
 }
 
-#[test]
```

Comment:
🪧 No logic changes here, I just moved the tests into a `#[cfg(test)]` module so they could share conditionally included imports.